primevue-gen/src/main.js

```diff
@@ -0,0 +1,22 @@
+import Vue from "vue"
+import PrimeVue from "primevue/config"
+import 'primevue/resources/themes/lara-light-teal/theme.css'
```

Comment:
Shouldn't we use the Bootstrap theme instead?

Reply:
@bai1024 , was that comment missed? I didn't see any code change to address it, or a reply to it.

Reply:
Sorry, this thread was hidden.
The components in design is based on `lara` theme instead of `Bootstrap` theme.

Reply:
Uhm..., maybe it was a misunderstanding from my side, but I thought the idea was to use the Bootstrap theme from Primevue because Promgen currently uses Bootstrap, so that would allow us to progressively change elements from the UI maintaining the current style, which would feel more natural.

Maybe we should discuss this withing the team.

Reply:
addressed this comments in [633b001](https://github.com/line/promgen/pull/456/commits/633b001ac6672e13beb2718892d50c126073216d)